scripts/gcp_setup/environment/gcloud_env.py

```diff
@@ -403,17 +404,20 @@ def has_permissions(self):
         User must be an org admin in order to assign a service account roles
         on the organization IAM policy.
         """
-        self._print_banner('Checking permissions')
+        if not self.skip_iam_check:
+            self._print_banner('Checking permissions')
 
-        if self._is_org_admin() and self._can_modify_project_iam():
-            print('You have the necessary roles to grant roles that Forseti '
-                  'needs. Continuing...')
+            if self._is_org_admin() and self._can_modify_project_iam():
+                print('You have the necessary roles to grant roles that '
```

Comment:
Had to reformat the `print` lines here to get the pylint check to pass. It didn't like the additional length that was caused by the extra level of indentation. No changes to the verbiage.